mslib/msui/mscolab.py

```diff
@@ -694,6 +698,26 @@ def after_login(self, emailid, url, r):
             self.signal_login_mscolab.emit(self.mscolab_server_url, self.token)
 
     def fetch_gravatar(self, refresh=False):
+        # Display custom profile picture if exists
+        from mslib.mscolab.server import APP
+        with APP.app_context():
+            fm = FileManager(APP.config["MSCOLAB_DATA_DIR"])
```

Comment:
this is broken by design. 

A user should be able to access a server which is not on his machine. 

Many users can share one server located at somewhere, you can not expect a local MSCOLAB_DATA_DIR has the same data for any user.

Reply:
you need a request over the server to fetch the image from a database on a remote mscolab server.